src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -207,8 +206,17 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
             foreach (var language in languages)
             {
                 notifier.NotifyCurrentProgress(string.Format(CultureInfo.CurrentCulture, Strings.DownloadingQualityProfileProgressMessage, language.Name));
+
+                QualityProfile qualityProfileInfo;
+                if (!host.SonarQubeService.TryGetQualityProfile(this.connectionInformation, this.project, language.ServerKey, cancellationToken, out qualityProfileInfo))
+                {
+                    failed = true;
+                    break;
```

Comment:
We verify the C# plugging during connect (before Bind). Either way I didn't change the flow, previously the same flow existed although a bit deeper so if there is no quality profile for a language we stop.